style.css

```diff
@@ -0,0 +1,3 @@
+.header {
+  color: green;
+}
```

Comment:
Please change either the `class name` here in CSS or give this `class name` for the `h1` element in your `index.html` file. Currently, the style is not appearing on the page.

Reply:
Done as advised